packages/web-client/tests/acceptance/deposit-test.ts

```diff
@@ -60,6 +60,9 @@ module('Acceptance | deposit', function (hooks) {
 
     assert.dom(post).containsText('Connect your L1 test chain wallet');
 
+    let finishedCssAnimation = false;
+    setTimeout(() => (finishedCssAnimation = true), 1000);
```

Comment:
Can we make the animation zero duration in test to avoid needing to introduce this delay and noise?